src/cartservice/src/Dockerfile

```diff
@@ -28,4 +28,5 @@ RUN GRPC_HEALTH_PROBE_VERSION=v0.4.11 && \
 WORKDIR /app
 COPY --from=builder /cartservice .
 ENV ASPNETCORE_URLS http://*:7070
+ENV DOTNET_EnableDiagnostics=0
```

Comment:
I think it's fine to disable `DOTNET_EnableDiagnostics`.
I don't believe anyone working on Online Boutique uses .NET debugging and profiling (on the container itself).

Reply:
Yeah, and I have read somewhere that anyway it's better for production workload to disable this anyway.